mx.py

```diff
@@ -750,6 +750,8 @@ def suite_context_free(func):
     Decorator for commands that don't need a primary suite.
     """
     _suite_context_free.append(func.__name__)
+    if '_' in func.__name__:
```

Comment:
We should make this a function to avoid repetition.

Reply:
Fixed